Documentation/api-reference.md

```diff
@@ -166,6 +166,56 @@ sendEvent: function(xdmData as object, callback = _adb_default_callback as funct
   end sub, context)
 ```
 
+#### Example of SendEvent with IdentityMap:
+
+> **Note**
+> `sendEvent` API allows passing custom identityMap with custom identifiers to the edge network. Create the map using identifier namespace and configure the "primary" and "authenticatedState" or the individual identity as per the application requirements.
+
+> **Note**
+> By Default `ECID` sent by the SDK is set as primary on the platform side if no other identifier is set as primary.
+
+> **Warning**
+> Do not pass ECID in this IdentityMap. ECID will be attached by the SDK automatically.
```

Comment:
- rewording suggestion + grouping with the note above:
Do not pass the ECID with the sendEvent API, the ECID automatically attaches it on all requests. By default, the ECID is set as primary server-side if no other identifier is set uses "primary" : true.
- move under the "Send custom IdentityMap" subtitle